### PR TITLE
Create PAF-Integration-Guide.md

### PR DESCRIPTION
**Pull request by BasileLeparmentier**
First draft, will improve uppon

### CODE REVIEW

File: mvp-spec/PAF-Integration-Guide.md [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+-   If the CMP is provided by a 3rd party partner, the Main Agreement between the Website and the CMP partner must also include the Model Terms.
+    
+-   Same applies if the PAF Client Node is provided to the Website by a 3rd party partner.
+    
+
+## Website Owner Tasks
+
+### Build the PAF components from sources (if needed)
+
+     mkdir paf-from-source
```
Comment (OlivierChirouze) [inline mvp-spec/PAF-Integration-Guide.md:82]: Here you _should_ use triple quotes to have syntax highlighting 😄
` ```shell`

File: mvp-spec/PAF-Integration-Guide.md [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+| Apply market research to generate audience insights 	| Included in Personalized Marketing<br>Included in Standard Marketing 	|
+| Develop and improve products                        	| Included in Personalized Marketing<br>Included in Standard Marketing 	|
+| Store and/or access information on a device         	| Included in Personalized Marketing<br>Included in Standard Marketing 	|
+| Ensure security, prevent fraud, and debug           	| Legimate interest, can’t be turned off                               	|
+| Technically deliver ads or content                  	| Legimate interest, can’t be turned off                               	|
+
+-   UI changes
+    
+
+![](https://api.media.atlassian.com/file/a2c9a4ce-b926-4722-bae6-b599b528ad68/image?allowAnimated=true&client=686c26b7-b3bd-47b6-a61b-c5b9817c3db9&collection=contentId-1725270776&height=125&max-age=2592000&mode=full-fit&token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI2ODZjMjZiNy1iM2JkLTQ3YjYtYTYxYi1jNWI5ODE3YzNkYjkiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpjb2xsZWN0aW9uOmNvbnRlbnRJZC0xNzI1MjcwNzc2IjpbInJlYWQiXX0sImV4cCI6MTY1MTY3ODQ5MywibmJmIjoxNjUxNjc1NTUzfQ.ZU38pLpbTRYn-LiIcrymiwrAoy164sE87Q-wKjoc_ng&width=156)
```
Comment (OlivierChirouze) [inline mvp-spec/PAF-Integration-Guide.md:356]: **must** be removed

File: mvp-spec/PAF-Integration-Guide.md [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+
+The PAF Client Node front end will be used by the prebid modules and the callback to get the id and preferences, generate seeds, and collate the transmissions into audit logs for each ad.
+
+The PAF client node is currently hosted by Criteo, but could be hosted by someone else.
+
+-   _[Ask Client node provider to]_ Add a new instance for the publisher website
+    
+    -   Configure **the server** to accept connections on the vhost of the subdomain the Website Owner allocated (e.g. `paf.example-website.com`)
+        
+        -   ![:warning:](https://pf-emoji-service--cdn.us-east-1.prod.public.atl-paas.net/atlassian/warning_64.png) the TLD+1 must be the same as the Website (ex: `www.example-website.com` and `some-sub-site.example-website.com` must use a PAF with domain `.example-website.com`)
```
Comment (OlivierChirouze) [inline mvp-spec/PAF-Integration-Guide.md:383]: remove this huge emoji

File: mvp-spec/PAF-Integration-Guide.md [hunk 4]
```diff
@@ -0,0 +1,663 @@
...
+            
+ **Configure** this new client. For example, using [the NodeJS implementation](https://github.com/prebid/paf-mvp-implementation/tree/main/paf-mvp-operator-client-express "https://github.com/prebid/paf-mvp-implementation/tree/main/paf-mvp-operator-client-express"):
+
+       // This is just an example of a basic client node configuration
```
Comment (OlivierChirouze) [inline mvp-spec/PAF-Integration-Guide.md:387]: use syntax highlight

` ```javascript `

**Review by RomainLofaso — APPROVED** [review]

**Pull request merged**